pkg/agent/openflow/client.go

```diff
@@ -1045,11 +1099,19 @@ func (c *client) ReplayFlows() {
 	}
 
 	for _, activeFeature := range c.activatedFeatures {
+		featureName := activeFeature.getFeatureName()
+		for _, meter := range activeFeature.replayMeters() {
+			// Openflow bundle message doesn't support meter. Add meter individually instead of
+			// calling AddOFEntries function.
```

Comment:
@wenyingd is this a limitation of ofnet / libOpenflow, or is this a limitation of OpenFlow?

Reply:
As far as I know, it's a limitation of OVS.